rl_utils/rl_utils/utils/reward.py

```diff
@@ -37,11 +36,15 @@ def __init__(
         self._curr_dist_to_path = None
         self.safe_dist = safe_dist
 
-        self.custom_rew_dict = custom_rew_dict
+        # load custom reward function from yaml
+        import tools.train_agent_utils as train_agent_utils
```

Comment:
Imports ganz oben in die File

Reply:
Circular import

Reply:
Man könnte die Train agent utils file noch ein bisschen auseinander nehmen und einzelne files erstellen. Bietet sich sowieso an weil eine 850 Zeilen Datei doch eher unübersichtlich ist.